### ISSUE TITLE
zprintrc with comments fails with 1.0.0 native image on linux

### ISSUE THREAD

**Issue by lvh** [kkinnear/zprint#151]
Downloaded zprintl for the 1.0.0 release off GitHub. Following zprintrc:

```clojure
{:map
 {:comma? false
  ;; lift-ns? will make deps.edn entries with just {:mvn/version "..."}
  ;; turn into #:mvn{:version "..."}, which is less legible.
  :lift-ns? false
  :key-order [:type :mvn/version :git/sha]}}
```

results in:

```
$ zprint          
Unable to read configuration from file /home/lvh/.zprintrc because java.lang.RuntimeException: EOF while reading, starting at line 1, compiling:(null:1:1)
```

Removing the comments, but otherwise keeping the zprintrc identical, works as expected. In order to check I'm not missing the obvious, I tried to use `jet < ~/.zprintrc` to parse the file and it works fine:

```clojure
{:map {:comma? false, :lift-ns? false, :key-order [:type :mvn/version :git/sha]}}
```

Maybe this is because we're still trying read-string on Graal and we're missing an exception case?

**Comment by kkinnear** [comment]
Thanks for noticing this. Near as I can tell, comments in a `.zprintrc` have never worked.   This was masked by a bug which I fixed in 0.5.4, issue #126. 

It was caused by the way that zprint reads the lines from the file and then reassembles them into a string. It left out the new lines during reassembly. 

This will be fixed in 1.0.1. 

Thanks again for letting me know about this!

**Comment by jhacksworth** [comment]
It took me some time to figure out why my custom settings weren't taking effect. Eventually, I realized it was the comments in the config that were causing the problem.

It might be worth mentioning in the readme that `.zprintrc` and `.zprint.edn` should not contain comments. I don't think it's mentioned anywhere, although I could be blind.

Besides that small issue, though, zprint is awesome. You've done an incredible job, thanks for sharing it! 🍻

**Comment by kkinnear** [comment]
Sorry this caused a problem for you too.  I never documented it because I didn't realize comments didn't work, never having tried it myself. I have fixed it so that comments are now supported in .`zprintrc` files and the fix will ship in the next release, which should be in the next few weeks.  

Thanks for the positive feedback.  I never really know if people are actually using zprint or how they feel about it except for the comments like yours.  It means a lot, so thanks again.

**Comment by jhacksworth** [comment]
> I have fixed it so that comments are now supported in .zprintrc files and the fix will ship in the next release, which should be in the next few weeks.

That is great news! Thank you!

>  I never really know if people are actually using zprint or how they feel about it

Yeah, I think open source projects could do with more positive feedback. People should know their work is appreciated.

**Comment by kkinnear** [comment]
This is now available in 1.0.1.  Thanks for letting me know about it!
